Framework/NexusGeometry/src/NexusGeometrySave.cpp.orig

```diff
@@ -0,0 +1,827 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
```

Comment:
You didn't mean to add this. Delete it, and use `git status` in future to check what you are staging before committing in git.

Reply:
done